website/docusaurus.config.js

```diff
@@ -26,7 +26,7 @@ const linksSocial = [
 const config = {
   title: 'OpenLineage',
   tagline: 'OpenLineage',
-  url: 'https://openlineage.io',
+  url: 'https://openlineage.github.io',
```

Comment:
Has no effect now, will be changed anyway in future PRs. 
Change under `website` directory was required to trigger website build process.